src/pyhf/modifiers/staterror.py

```diff
@@ -31,10 +31,11 @@ def __init__(self, staterr_mods, pdfconfig, mega_mods, batch_size=None):
         keys = ['{}/{}'.format(mtype, m) for m, mtype in staterr_mods]
         staterr_mods = [m for m, _ in staterr_mods]
```

Comment:
fix this up. use self._staterr_mods and remove the internal/temp variable probably?

Reply:
done